src/features/recruitments/components/condition-filter.tsx

```
@@ -54,36 +46,27 @@ const getActiveCount = (
-        적용하기
-      </button>
-    </div>
+    <button
+      onClick={onClick}
+      className={cn(
+        'cursor-pointer rounded-full border px-3.5 py-1.5 text-sm font-medium transition-colors',
+        selected
+          ? 'bg-primary/10 text-primary border-transparent'
+          : 'border-border bg-background text-foreground hover:bg-secondary',
+      )}
+    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)